### DISCUSSION TITLE
Superscript in markdown table columns

### DISCUSSION THREAD

**Discussion by beingalink** [sqlpage/SQLPage#729] [Q&A] [answered]
Hi all,

I'm just trying to use markdown columns for being able to display superscript (i.e. 1.345<sup>p,q</sup>). Unfortunately, I can't figure it out as neither `^p,q^` nor `<sup>p,q</sup>` seem to work. Am I missing something? Thanks for your help in advance!

**Comment by lovasoa** [comment, marked as answer]
superscript is unfortunately not supported in the markdown library we currently use

**Comment by lovasoa** [comment]
I opened a separate feature request to track this: https://github.com/sqlpage/SQLPage/issues/732

**Comment by beingalink** [comment]
Thank you!
